Modules/zipimport.c

```diff
@@ -379,21 +405,29 @@ find_loader(ZipImporter *self, PyObject *fullname, PyObject **namespace_portion)
     return FL_MODULE_FOUND;
 }
 
+/*[clinic input]
+zipimport.zipimporter.find_module
+
+    fullname: unicode
```

Comment:
Actually *all* the methods here were positional-only. But this change is backwards compatible and logical. See for example http://bugs.python.org/issue20294#msg208444
Why should we force positional-only? Your patch in bpo-25711 to convert `zipimport` to Python would also have the same effect.